CHANGELOG.md

```diff
@@ -0,0 +1,91 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [5.0.2] - 2023-03-20
+
+### Added
+
+-   Better support for browser packaging
+-   Generating independent browser js files
+
+## [5.0.1] - 2023-02-23
+
+### Fixed
+
+-   Fixed issue with metadata endpoint in browsers by @sagivf
```

Comment:
Do we note people in release notes :)

Reply:
I try to mention outside contributors/people who contributed for the first time